src/main/java/frc/robot/Constants.java

```diff
@@ -11,164 +11,198 @@
+    /* Back Right Module - Module 3 */
+    public static final class Mod3 { //TODO: This must be tuned to specific robot
+      public static final int driveMotorID = 41;
+      public static final int angleMotorID = 42;
+      public static final int canCoderID = 4;
+      public static final Rotation2d angleOffset = Rotation2d.fromDegrees(231.503);
+      public static final SwerveModuleConstants constants = new SwerveModuleConstants(driveMotorID, angleMotorID,
+          canCoderID, angleOffset);
     }
+  }
 
-    // Create constants for the Arm subsystem
-    public static final class ArmConstants {
-        public static final int topMotorID = 1;
-        public static final int bottomMotorID = 9;
+  public static final class AutoConstants { //TODO: The below constants are used in the example auto, and must be tuned to specific robot
+    public static final double kMaxSpeedMetersPerSecond = 11;
+    public static final double kMaxAccelerationMetersPerSecondSquared = 3;
+    public static final double kMaxAngularSpeedRadiansPerSecond = Math.PI;
+    public static final double kMaxAngularSpeedRadiansPerSecondSquared = Math.PI;
 
-        public static final int innerLimitSwitchID = 9;
-        public static final int outerLimitSwitchID = 8;
+    public static final double kPXController = 1;
+    public static final double kPYController = 1;
+    public static final double kPThetaController = 1;
 
-        public static final double armSpeed = 0.15;
+    /* Constraint for the motion profilied robot angle controller */
+    public static final TrapezoidProfile.Constraints kThetaControllerConstraints = new TrapezoidProfile.Constraints(
+        kMaxAngularSpeedRadiansPerSecond, kMaxAngularSpeedRadiansPerSecondSquared);
+  }
 
-        public static final int midArmPosition = -2500;
-        public static final int hybridArmPosition = -3240;
+  // Create constants for the Arm subsystem
+  public static final class ArmConstants {
+    public static final int topMotorID = 1;
+    public static final int bottomMotorID = 9;
 
-    }
+    public static final int innerLimitSwitchID = 9;
+    public static final int outerLimitSwitchID = 8;
 
-    // Should log to USB
-    public static final boolean shouldLogToUSB = false;
+    public static final double armSpeed = 0.15;
 
-    // Controller Constants
-    public static final class Controllers {
+    public static final int midArmPosition = -2500;
+    public static final int hybridArmPosition = -3240;
 
-        public static final int driverController = 0;
-        public static final int operatorController = 1;
+  }
 
-    }
+  // Should log to USB
+  public static final boolean shouldLogToUSB = false;
+
+  // Controller Constants
+  public static final class Controllers {
+
+    public static final int driverController = 0;
+    public static final int operatorController = 1;
+
+  }
+
+  public static class PneumaticConstants {
```

Comment:
Please verify constants naming with **camelCase** and naming ex (drawerForwardChanel -> drawerForwardID)